src/eei.h

```diff
@@ -116,8 +116,28 @@ struct EthereumInterface : ShellExternalInterface {
 };
 
 struct GasSchedule {
+  static constexpr unsigned storageLoad = 200;
   static constexpr unsigned storageStoreCreate = 20000;
   static constexpr unsigned storageStoreChange = 5000;
+  static constexpr unsigned log = 375;
+  static constexpr unsigned logData = 8;
+  static constexpr unsigned logTopic = 375;
+  static constexpr unsigned create = 32000;
+  static constexpr unsigned call = 700;
+  static constexpr unsigned copy = 3;
+  static constexpr unsigned blockhash = 800;
+  static constexpr unsigned balance = 400;
+  static constexpr unsigned stepGas0 = 0;
+  static constexpr unsigned stepGas1 = 2;
```

Comment:
I think I'd prefer having named gas fields for each of the interface method as opposed to YellowPaper-esque `stepGas*`

Reply:
any recommendations for a naming scheme?

Reply:
Probably the name of the interface method?

Reply:
Lots of these are used for multiple interface methods.

Reply:
I'd suggest to with the YP in that case: Appendix G of https://ethereum.github.io/yellowpaper/paper.pdf